docs/design/liqonet_tunnelEndpoint.md

```diff
@@ -0,0 +1,17 @@
+# TunnelEndpoint-Operator
+## Overview
+This component is in charge of establishing a VPN tunnel with the peering clusters, which is created between the (local) Gateway Node and the one associated to the foreign cluster.
+All the traffic between the local and remote cluster is first delivered to the Gateway Node, then tunneled towards the remote gateway and finally delivered to the desired destination.
+The traffic can leave local cluster as it is in case the home and remote addressing spaces do not overlap; otherwise, the traffic crosses a properly configurated NAT in order to avoid overlapped spaces.
+
+The TunnelEndpoint-Operator runs as deployment only on the local Gateway Node.
+
+### Features
+* GRE tunnel as VPN tunnel
+
+### Limitations
+* Does not support other VPN solutions
```

Comment:
So, it means that supports only GRE tunnels?

Reply:
yes, only GRE tunnels.